manifests/s/Sonos/Controller/70.1.34112/Sonos.Controller.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Sonos.Controller
+PackageVersion: 70.1.34112
+PackageLocale: en-US
+Publisher: Sonos, Inc.
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+Author: Sonos, Inc.
+PackageName: Sonos
+PackageUrl: https://support.sonos.com/s/downloads?language=en_US
+License: Copyright (c) Sonos Inc.
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate